traffic_prophet/config.py

```diff
@@ -11,5 +11,15 @@
                         1147135],
     'exclude_ptc_pos': [446402, 7204532, 1145377, 30029635, 1147358, 106853,
                         1140996, 1797, 14177830, 30073989, 14189397,
-                        440428, 14659261]
+                        440428, 14659261],
+    'n_neighbours': 5,
+    'neighbour_distance_metric': 'Euclidean'
```

Comment:
It's not like we're currently using this - delete until we have a wrapper class.